### DISCUSSION TITLE
What is the purpose of preferred scales?

### DISCUSSION THREAD

**Discussion by TheKorn2** [zwave-js/zwave-js-ui#3588] [Q&A] [answered]
What is the purpose of preferred scales in the config panel?   Reason I ask is because my preferred units are F, and I've set them there.  But some of my sensors apparently only return C, and so I have to manually convert them.

If automatic unit conversions aren't on the table, why are we asked about preferred scales at all?

Thanks!

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by AlCalzone** [comment, marked as answer]
If a device supports multiple scales (C and F for example), Z-Wave JS doesn't know which one to use during the interview (and the fallback polls every few hours), because it has to tell the device in which scale to answer.
Without this setting, you get the first supported one (which would be C in this case).

**Comment by TheKorn2** [comment]
Fair enough!
